packages/storage/src/common/cid.ts

```diff
@@ -1,8 +1,8 @@
-import { FileOrBufferOrString } from "../types";
-import { DEFAULT_GATEWAY_URLS } from "./urls";
-import { isBufferOrStringWithName } from "./utils";
+import {FileOrBufferOrString} from "../types";
```

Comment:
prettier will likely complain about this, why the formating change?

Reply:
Nice catch, fixed!